app/domain/etl/ga/views_and_navigation_service.rb

```diff
@@ -39,6 +38,10 @@ def append_data_labels(values)
     }
   end
 
+  def long_page_path?(data)
+    data['page_path'].length > PAGE_PATH_LENGTH_LIMIT
+  end
```

Comment:
I was thinking more along the lines of rejecting anything with query params, e.g.

```ruby
!URI.parse(data['page_path']).query.nil?
```

That's a bit more explicit with respect to our intent, but I'm fine with this approach as well.